### ISSUE TITLE
Issues displaying svg prop that's not located in the same folder

### ISSUE THREAD

**Issue by KevinGomezDev** [Agezao/confetti-generator#12]
Hello @Agezao Thank you for this wonderful plugin.

I've been trying to use the new svg prop functionality to include a new image but every-time that I added a svg attribute to the settings payload I'm getting the following error:

> Uncaught TypeError: Failed to execute 'drawImage' on 'CanvasRenderingContext2D': The provided value is not of type '(CSSImageValue or HTMLImageElement or SVGImageElement or HTMLVideoElement or HTMLCanvasElement or ImageBitmap or OffscreenCanvas)'

Seems like a problem with the drawImage function like if the image is invalid, I've tried different formats including, png, jpg and svg but all of them triggered the same issue

https://github.com/Agezao/confetti-js/blob/master/src/confetti.js#L137

Usage: 
![Screen Shot 2019-11-13 at 12 49 56 PM](https://user-images.githubusercontent.com/25872784/68789795-22223080-0614-11ea-8ad0-f9e14258d97c.png)

Any idea?

**Comment by Agezao** [comment]
Hi.

Here it worked fine with your svg file:
![image](https://user-images.githubusercontent.com/7091570/68794122-23ac2280-062d-11ea-9eb0-5d6f61fbd51e.png)

I would suggest you to return a working, fixed/mocked, json at `defaultSettings` and start to forward-engineer* from this point, adding the other props bit-by-bit.

**Comment by KevinGomezDev** [comment]
I've tried the plain settings Mock but it's returning the same warning.

@Agezao Have you tried the plugin on react?
I've found something interesting that's probably the cause of the issue that I'm currently having on my react implementation:

In your example located on https://agezao.github.io/confetti-js/
new Image() is generating a DOM img element as you can see in the following screenshot
![Screen Shot 2019-11-13 at 4 22 09 PM](https://user-images.githubusercontent.com/25872784/68805582-1cd3de80-0632-11ea-954d-90c7f1aebc8e.png)

But In My current React Implementation seems like `new Image()` is creating a React Adapter() with props, context, updaters etc, Like if it's a react element.
![Screen Shot 2019-11-13 at 4 23 30 PM](https://user-images.githubusercontent.com/25872784/68805666-47259c00-0632-11ea-8679-0c7a451ff1e1.png)

**Comment by Agezao** [comment]
I can't see how you are importing and using `confetti-js` in your project, but, I and other users have successfully implemented this lib in react before.

Did these implementations looks similar to yours?
https://github.com/Agezao/confetti-js/issues/8#issuecomment-435735714
https://github.com/Agezao/confetti-js/issues/1#issuecomment-392802232

Is it next.js though? I've never personally implemented confetti-js in a next.js project _(although I think this would not influence how it works)_

**Comment by KevinGomezDev** [comment]
I think I've figured the issue, I'm using `webpack (ProvidePlugin)` to allow globally imports of my own `Image` Component and it's affecting the Confetti Plugin (`new Image()` method) as well which makes the `Image` a React Component instead of a DOM Element, and that will trigger the issue, because the `ctx.drawImage` can not handle React Elements.

Thank you so much for the help @Agezao.
Closing the Ticket.
